### DISCUSSION TITLE
How to update property value from front end graph to server model

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-glsp/glsp#907] [Q&A] [answered]
Hi 
   I want to know when I change the model property on the front graph, How could I update the value to the model? and by what kind of action or operation. like follow picture shows, I changed the name of a node, and I want to update the server model, I do not know how to implement this. I have thought to create a custom action, but is there any built-in method to do this?
![image](https://user-images.githubusercontent.com/4537809/217202908-2fb727dd-4f44-4d31-9ce5-cb15bf66dd6c.png)

**Comment by tortmayr** [comment, marked as answer]
HI @wss29 ,
 Typically model property updates are highly domain specific and require some sort of customization  anyways so we currently don't offer a generic mechanism for this. 
So using a custom action  is the recommended approach here. For your usecase, you should probably implement a custom  update[`Operation`](https://www.eclipse.org/glsp/documentation/modeloperations/) and implement the corresponding `OperationHandler`.
